CHANGELOG.md

```diff
@@ -8,8 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Dependencies
 
-* Update AWS SDK for S3 from 0.17 to 0.19
-* Update AWS SDK common crates like `aws-config`, `aws-http`, etc from 0.47 to 0.49
+* Update AWS SDK for S3 from 0.17 to 0.22
```

Comment:
I thought it will be odd if we'll have two lines, one says that we updated from 0.17 to 0.19, and another one from 0.19 to 0.22, but okay)

I have read instruction how to contribute to ssstar https://github.com/elastio/ssstar/blob/master/CONTRIBUTING.md

I think it is needed because it's open source project

Reply:
fixed